packages/frontend/web/components/lib/ArticleRenderer.tsx

```diff
@@ -1,10 +1,16 @@
 import { TextBlockComponent } from '@frontend/web/components/elements/TextBlockComponent';
 import { ImageBlockComponent } from '@frontend/web/components/elements/ImageBlockComponent';
 import React from 'react';
-
+// import { clean } from '@frontend/model/clean';
 export const ArticleRenderer: React.FC<{ elements: CAPIElement[] }> = ({
     elements,
 }) => {
+    // const cleanedElements = elements.map(element =>
+    //     'html' in element ? { ...element, html: clean(element.html) } : element,
+    // );
+    // ^^ Until we decide where to do the "isomorphism split" in this this code is not safe here.
+    //    But should be soon.
+
```

Comment:
I might be missing some context here, but why would this be unsafe here now? And why would it be safe soon?

Reply:
Because `ArticleRenderer.tsx` runs in the current client bundle and now has a dependency on jsdom. If/when we don't use React in the client bundle then this doesn't matter. OTOH it might be better to put it back in extractcapi